backend/src/main/kotlin/me/andrewda/controllers/RequestController.kt

```diff
@@ -20,6 +21,21 @@ object RequestController {
         }
     }
 
+    suspend fun createSeveral(requests: List<NewRequest>, person: Person) = query {
+        requests.map {
+            if (it.item == null) throw MissingFields()
+
+            Request.new {
+                personId = person.id
+                itemId = EntityID(it.item, Items)
+
+                if (it.quantity != null) {
```

Comment:
This is redundant, `it.quantity` will never be null. We should make `it.quantity` non-optional anyway since `NewRequest.isValid` mandates `quantity` is non-null.